### ISSUE TITLE
[eslint-plugin-lit-a11y] change default to not check lit-html imports by default

### ISSUE THREAD

**Issue by thepassle** [open-wc/open-wc#1934]
# TL;DR:
We should make it the default to always lint any html tagged template literal. If people use multiple rendering libraries in their projects, they should opt in to only check lit-html imported tagged template literals, by doing either: `"litHtmlSources": true` or `"litHtmlSources": ["ing-web"]` in the eslint settings**

## Details:
We currently are able to check if a html tagged template literal is actually a lit-html tagged template literal by checking the imports

We have several options regarding this:
- By default, we allow lit-html to only be imported from `lit-html` and `lit-element`, but it could be the case that a package re-exports lit-html, like `ing-web`. In this case, `ing-web` can be added as a setting as “valid lit-html import”, like so: `"litHtmlSources": ["ing-web"]`
- We can disable checking for a valid lit-html source, and just lint any html tagged template literal, like so: `"litHtmlSources": false`

The reason we have the check for a valid lit-html import, is because people may use multiple rendering libraries in a project, like both `lit-html` and `htm` in the same project. We only want to lint the `lit-html` tagged template literals in that case, and not `htm` templates because they use different syntax

The default we currently have is to only lint html tagged template literals imported from either `lit-html` or `lit-element`. If people dont import from `lit-html` or `lit-element`, they have to add an option:

```json
  "eslintConfig": {
    "extends": [
      "eslint-config-ow",
      "plugin/lit-a11y:recommended"
    ],
    "settings": {
      "litHtmlSources": ["ing-web"]
    }
  },
```

I think we should make the default to completely disable checking for a lit-html import, then implementing projects only have to do:

```json
  "eslintConfig": {
    "extends": [
      "eslint-config-ow",
      "plugin/lit-a11y:recommended"
    ],
}
```

People who use multiple rendering libraries in their projects should opt-in to the behavior of only checking lit-html tagged template literals

I think disabling the lit-html import makes it easier for more people to add the plugin to their project, than there are people really using multiple rendering libraries in their project

## Todo:
- [ ] Make linting _any_ html tagged template literal the default
- [ ] Update the docs in the packages main `README.md`

**Comment by thepassle** [comment]
this was fixed in #1936
